cidc_api/models/files/facets.py

```diff
@@ -247,9 +264,12 @@ def __init__(self, match_clauses: List[str], description: Optional[str] = None):
     ),
 }
 
+analysis_ready_facets = {"Olink": FacetConfig(["npx|analysis_ready|csv"])}
```

Comment:
I'm not sure about adding this entirely separate facet category - maybe we should just leave this file available in the `"Olink" > "Study-Level"` facet, with a plan to discuss as a team how we want to better surface and describe CIDC-generated derivative files in the portal?

Reply:
it's also there, this was just a suggestion that I had as a good place to put this saliently for the users

Reply:
Yeah, on second thought, let's just go for this. Better to put it somewhere people will stumble across it and make changes if necessary based on their feedback.